src/solvers/refinement/string_refinement.h

```diff
@@ -77,11 +67,9 @@ class string_refinementt final: public bv_refinementt
   string_refinementt(const infot &, bool);
   bvt convert_bool_bv(const exprt &boole, const exprt &orig);
 
-  unsigned initial_loop_bound;
-
+  const configt config_;
+  unsigned loop_bound_;
```

Comment:
use size_t
no underscore suffixes

Reply:
But we already agreed on underscore suffixes ;(

Reply:
I think we should at least consider underscore suffixes for data members in new code. It makes it much easier to differentiate data members from global data or method parameters.